src/manager.ts

```diff
@@ -998,7 +998,10 @@ class Manager<
         );
     };
 
-    public _createCustomFilteredQueryRequest = (
+    /**
+     *  Get the raw ES query based on the current state.
+     */
+    public getEsQuery = (
```

Comment:
can we call this method `generateESRequestFromCurrentFilterState` or something a little more explicit?

Reply:
Oh @markrsocialnative can you also document this method in the README?

Reply:
yep